code/game/objects/items/stacks/rods.dm

```diff
@@ -21,8 +21,11 @@
 		if(amount < 8)
 			to_chat(user, span_warning("You need at least [8 - amount] more [src] to make razorwire obstacles!"))
 			return
+		if(B.amount < 2)
+			to_chat(user, span_warning("You need [2 - B.amount] more [B] to make a razor wire obstacle!"))
+			return
 		use(8)
-		B.use(1)
+		B.use(2)
 		var/obj/structure/razorwire/M = new /obj/item/stack/razorwire(user.loc, 2)
 		to_chat(user, span_notice("You combine the rods and barbed wire into [M]!"))
 
```

Comment:
```suggestion
		to_chat(user, span_notice("You combine the rods and barbed wire into [M]!"))
		return
```

Reply:
imma be honest you are more code litterate in DM than me so imma trust you on that return

Reply:
So this is what's called an early return. Since you did everything you wanted to in the attackby proc, you return out as soon as you can so unnecessary stuff you don't want or need to do doesn't happen. If you look down in the iswelder block below it, you'll see the same thing for the same reason.

Reply:
ah I see should propably do the same to the barbed wire then